src/modules/core/components/Button/Button.tsx

```diff
@@ -44,7 +44,9 @@ export interface Props
   disabled?: boolean;
 
   /** Pass a ref to the `<button>` element */
-  innerRef?: (ref: HTMLElement | null) => void;
+  innerRef?:
+    | ((ref: HTMLElement | null) => void)
+    | React.MutableRefObject<HTMLButtonElement | null>;
```

Comment:
Updating the component to allow the passing of an actual ref object (used by the color picker), not just the function you receive when using the popover component